Bottom: 87423f82c99520be60821443978acef12f9203b0
Top:    ddfb232cd8662ca03dc1fe9224976120a33be45f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 00:29:41 +0000

Refresh of hid-gtk-disable-gl-alpha-blend

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 51bdffa..ace9851 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1084,9 +1084,10 @@ ghid_pinout_preview_expose (GtkWidget *widget,
   hidgl_start_render ();
 
 #if 0  /* We disable alpha blending here, as hid_expose_callback() does not
-          call set_layer() as appropriate for us to sub-composite rendering
-          from each layer. If we leave alpha-blending on, it means text and
-          overlapping pads are rendered looking very ugly.
+        * call set_layer() as appropriate for us to sub-composite rendering
+        * from each layer. If we leave alpha-blending on, it means text and
+        * overlapping pads are rendered looking very ugly.
+        */
 
   glEnable (GL_BLEND);
   glBlendFunc (GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
